rules/windows/registry/registry_set/registry_set_potential_oci_dll_redirection.yml

```
@@ -0,0 +1,34 @@
+title: Registry Modification for Potential OCI DLL Redirection
```

You cannot use potential in high rules.
```suggestion
title: Registry Modification for OCI DLL Redirection
```